docs/UsersGuide/best/hub-pattern.md

```diff
@@ -17,4 +17,134 @@ ARINC 653 Channels, High-speed hardware buses between nodes, and UARTs between n
+
+If telemetry and event connections are tied to the hub, the setup order in setupTopology in obcBTopology.cpp will need to be slightly 
+reordered. This should be a swap between `configureTopology()` and `regCommands()`, which should match the following:
+```cpp
+// In Node B's Topology.cpp
+void setupTopology(const TopologyState& state) {
+    ... 
+    configureTopology();
+    regCommands();
+    ...
+} 
+
+```
+
+## Creating a Combined Dictionary for a Multi-deployment System
+
+Since this system is running on two deployments, the dictionaries would need to be merged together in order to process the data from both 
+deployments. Running `fprime-gds` using a dictionary from build-artifacts of one deployment but not both would drop telemetry and logging from 
+the other deployment, and GDS will not display any inner-deployment commands. Using [FPP Tools](https://nasa.github.io/fpp/fpp-users-guide.html#Specifying-Models-as-Files_Computing-Dependencies), we can generate a list of dependencies for 
+topologies of both deployments and create a combined dictionary.
+
+Create a new directory structure on the same level as the two other deployments. Additionally, create a new 
+topology.fpp file and a shell script file within the directory. The directory structure should look like this, 
+assuming that "obcA" and "obcB" are two deployments implementing the hub pattern.
+
+```
+FPrimeProject
+├── build
+├── build-artifacts
+├── ...
+├── obcADeployment
+├── obcBDeployment
+└── GDSDictionary
+  ├── generate_dictionary.sh
+  └── topology.fpp
+```
+
+In the new topology.fpp file, import the topologies from the two deployments like this:
+
+```shell
+# In GDSDictionary/Topology.fpp
+topology GDSDictionary {
+  import obcA.obcA
+  import obcB.obcB 
+} 
+```
+
+Using FPP-Tools, we can manually generate a ground dictionary by finding the dependencies of GDSDictionary, which includes the dependencies of 
+Node A and Node B. 
+`fpp-depend` writes out the names of generated files during dependency analysis. This is useful in retrieving the files needed to build the 
+dictionaries for both Node A and Node B. It also depends on the locs.fpp file generated
+when running `fprime-util generate`.
+`fpp-to-dict` is then used to retrieve the dependencies and build a dictionary.
+These can be either ran in the terminal or executed with a shell script.
+
+```shell
+#!/bin/bash
+
+fpp-depend ../build-fprime-automatic-native/locs.fpp topology.fpp -g deps.txt
+tr '\n' ',' < deps.txt | sed 's/,$//' > deps-comma.txt
+fpp-to-dict -i `cat deps-comma.txt` topology.fpp
+```
+> The second line takes the file dependencies and converts it to a one-line comma-separated input for `fpp-to-dict`. This line maybe omitted, 
+which `fpp-to-dict` would then generate dictionaries for the GDS and both nodes.
+
+## Reference
```

Comment:
I might recommend moving the example link up near the start so folks don't miss the complete example